packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -32,8 +34,12 @@ const showNetworkPage = false;
 /// Top-level configuration for the app.
 @immutable
 class DevToolsApp extends StatefulWidget {
+  const DevToolsApp([this.conditionalScreens]);
+
+  final List<ConditionalScreen> conditionalScreens;
+
   @override
-  State<DevToolsApp> createState() => DevToolsAppState();
+  State<DevToolsApp> createState() => DevToolsAppState(conditionalScreens);
```

Comment:
To make this code consistent with Flutter, don't pass this into the state constructor. Instead grab it off of the widget within the DevToolsAppState object.